frontend/app/routes/_index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Link } from '@remix-run/react';
+
+import { ButtonLink } from '~/components/buttons';
+import type { RouteHandleData } from '~/utils/route-utils';
+
+export const handle = {
+  documentTitleI18nKey: 'gcweb:header.application-title',
+} as const satisfies RouteHandleData;
+
+export default function RootIndex() {
+  return (
+    <>
+      <main role="main" className="flex h-screen bg-splash-page bg-cover bg-center" property="mainContentOfPage">
+        <div className="m-auto aspect-square w-[300px] md:w-[400px] lg:w-[500px]">
+          <div className="bg-white p-8">
+            <h1 className="sr-only">Canadian Dental Care Plan | Régime canadien de soins dentaires</h1>
+            <div className="w-11/12 lg:w-8/12">
+              <Link to="https://www.canada.ca/en.html" property="url">
+                <img className="h-8 w-auto" src={`/assets/sig-blk-en.svg`} alt="Government of Canada" property="logo" width="300" height="28" decoding="async" />
+              </Link>
+              <span className="sr-only">
+                {' '}
```

Comment:
Since this is `sr-only`, the forced-space (`{' '}`) isn't necessary. You can just remove it.

Reply:
Implemented in commit https://github.com/DTS-STN/canadian-dental-care-plan/commit/a4e03b4e0a9c0a1f4e8aa3712b0562864682aafe.